src/js/letters/reducers/index.js

```diff
@@ -90,8 +91,14 @@ function letters(state = initialState, action) {
       const benefitInfo = action.data.data.attributes.benefitInformation;
       const possibleOptions = [];
       Object.keys(benefitInfo).forEach(key => {
-        if ((optionsToAlwaysDisplay.includes(key) || (benefitInfo[key] !== false)) &&
-            !possibleOptions.includes[key]) {
+        if (
+          // the option should always be displayed or vets-api says it is available
+          (optionsToAlwaysDisplay.includes(key) || (benefitInfo[key] !== false))
+          // and the option is not yet in the possibleOptions array
+          && !possibleOptions.includes[key]
+          // and the option is a customization option that vets-api supports
+          &&  REQUEST_OPTIONS[key]
```

Comment:
This is the smallest of nits, but it looks like there might be an extra space after `&&`?

Reply:
ahh you're right, glad you found it :)
Kind of surprised that the linter didn't yell at me.